frontend/src/features/mapSideBar/SidebarContents/subforms/ParcelIdentificationForm.tsx

```diff
@@ -77,107 +83,139 @@ export const ParcelIdentificationForm: React.FC<IIdentificationProps> = ({
+              if (pin > 0) {
+                return pin;
+              }
+              return '';
+            }}
+            type="number"
+          />
+          <SearchButton
+            onClick={(e: any) => {
+              e.preventDefault();
+              handlePinChange(getIn(formikProps.values, withNameSpace('searchPin')), nameSpace);
+            }}
+          />
+        </Form.Row>
+        <Form.Row>
+          <Label>Street Address</Label>
+          <GeocoderAutoComplete
+            value={getIn(formikProps.values, withNameSpace('searchAddress'))}
+            field={withNameSpace('searchAddress')}
+            onSelectionChanged={selection => {
+              formikProps.setFieldValue(withNameSpace('searchAddress'), selection.fullAddress);
+              setGeocoderResponse(selection);
+            }}
+            onTextChange={value => formikProps.setFieldValue(withNameSpace('searchAddress'), value)}
+            error={getIn(formikProps.errors, withNameSpace('searchAddress'))}
+            touch={getIn(formikProps.touched, withNameSpace('searchAddress'))}
+            displayErrorTooltips
+          />
+          <SearchButton
+            onClick={(e: any) => {
+              e.preventDefault();
+              geocoderResponse && handleGeocoderChanges(geocoderResponse, nameSpace);
+            }}
+          />
+        </Form.Row>
+      </Col>
+      <Col md={1}>
+        <h5>OR</h5>
+      </Col>
+      <Col md={5} className="instruction">
+        <p>
+          Click on the pin, and then click your desired location on the map to pull the parcel
+          details.
+        </p>
+        <Row>
+          <Col className="marker-svg">
+            <SearchMarkerButton
               onClick={(e: any) => {
+                setMovingPinNameSpace(nameSpace ?? '');
                 e.preventDefault();
-                geocoderResponse && handleGeocoderChanges(geocoderResponse, nameSpace);
               }}
-            />
-          </Form.Row>
-        </Col>
-        <Col md={1}>
-          <h5>OR</h5>
-        </Col>
-        <Col md={5} className="instruction">
-          <p>
-            Click on the pin, and then click your desired location on the map to pull the parcel
-            details.
-          </p>
-          <Row>
-            <Col className="marker-svg">
-              <SearchMarkerButton
-                onClick={(e: any) => {
-                  setMovingPinNameSpace(nameSpace ?? '');
-                  e.preventDefault();
-                }}
-              >
-                <ParcelDraftIcon className="parcel-icon" />
-              </SearchMarkerButton>
-            </Col>
-          </Row>
-        </Col>
+            >
+              <ParcelDraftIcon className="parcel-icon" />
+            </SearchMarkerButton>
+          </Col>
+        </Row>
+      </Col>
+    </>
+  );
+
+  return (
+    <Container>
+      <Row>
+        <h4>Parcel Identification</h4>
       </Row>
+      {!disabled && (
```

Comment:
don't display either the search component or move pin component if the form is disabled.